compiler/exe4j/compiled-with-exe4j.yml

```diff
@@ -0,0 +1,19 @@
+rule:
+  meta:
+    name: compiled with exe4j
+    namespace: compiler/exe4j
+    author: johnk3r
+    scope: file
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information::Compile After Delivery [T1027.004]
+    examples:
+      - 6b25f1e754ef486bbb28a66d46bababe:0x404EDE
+  features:
+    - or:
+      - string: /exe4j_log/i
+      - string: /install4j_log/i
+      - string: /exe4j_java_home/i
+      - string: /install4j/i
+      - string: /exe4j.isinstall4j/i
+      - string: /com/exe4j/runtime/exe4jcontroller/i
+      - string: /com/exe4j/runtime/winlauncher/i
```

Comment:
i *think* we can use string literals (not regex) for these. did this not work for you?

we should prefer string literals because they will match faster.

Reply:
I didn't know that (match faster), thanks for reporting. So, with regex the result was more accurate. For this sample, the first session is with regex, the second with literal string. What is your opinion? 

![regex](https://user-images.githubusercontent.com/6247648/108598572-1ec83980-736d-11eb-9f0f-28eb82156e4d.png)

Reply:
If they're always upper or lower case, changing accordingly (adding both variants) would work. If there's more combination, e.g. `Exe4j`, `EXE4j`, ... I'd prefer the regex.

Reply:
Ok. I added the words in uppercase and regex just where it was really needed.